### ISSUE TITLE
Schedule periodical CI runs for master branches

### ISSUE THREAD

**Issue by santisoler** [fatiando/continuous-integration#18]
We have recently experienced an error on the master branch of Rockhound (https://github.com/fatiando/rockhound/issues/53) due to a problem after one of the dependencies was updated.
In this case, it was caught by a user, but it could also be caught by any contributor when opening a PR and the CI fails. This means that new errors on master branches are currently being caught after days they appear on situations where we don't want these errors to be caught (users, new PRs, etc).

Would be nice to schedule periodical CI runs for the master branch of all Fatiando libraries. This way we could caught this errors before any user encounter it, or any PR is delayed because of it.
Also, would be nice to program the @fatiando-bot to automatically open an issue on the corresponding repository if the scheduled CI run fails.

**Comment by leouieda** [comment]
This part of the configuration isn't part of these scripts and needs to be implemented in each repository. But it would also be nice if this repository had a scheduled build.

**Comment by leouieda** [comment]
I don't know if the bot can do it because there are some differences between the CI config files in each repo.

**Comment by santisoler** [comment]
I'm aware that scheduling periodical builds it's part of the CI configuration, but I thought this would be the best place to open this issue. Maybe the bot raising an issue is not the best idea. Is there a way to publish a failed scheduled build on Slack? My idea is to have a way to be notified of such failure on time.

**Comment by leouieda** [comment]
Right, there probably is. We can setup Slack integration on Travis (yes) and Azure (maybe) and they will post to `github_activity` when the build fails or something.
